manifests/supervisorcluster/1.24/cns-csi.yaml

```diff
@@ -425,7 +425,6 @@ data:
   "csi-sv-feature-states-replication": "true"
   "fake-attach": "true"
   "async-query-volume": "true"
-  "improved-csi-idempotency": "true"
```

Comment:
@chethanv28 can we remove these gates from the WCP deployment manifest files?
https://github.com/kubernetes-sigs/vsphere-csi-driver/search?q=improved-csi-idempotency

Do you see any implications when this gets spec bumped?

Reply:
Just want to make sure, if only yaml change gets picked up and code change is not picked up on the release branch, we might disable the feature accidentally.

Reply:
We can remote the gates on WCP. I don't see any issues there.